src/clonepi

```diff
@@ -1091,6 +1091,17 @@ runPostSyncHook()
 	fi
 }
 
+initDest()
+{
+sfdisk --delete /dev/$DEST_DISK
+echo -e "size=+63M, type=e\ntype=5\nsize=+32M,type=83\nsize=+256M,type=c\ntype=83"| sfdisk $DEST_DISK
```

Comment:
This is very specific structure, that matches your current install & will likely match most with NOOBS, but won't match many others.

This is the real reason ClonePi currently does a partial dd to copy the MBR - there's a lot of ways to setup a system at the partition level.